Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
We can't do this in this method, because it is a BC break. So we must handle it in the calling method in `ColorSelectorType`.